src/libs/actions/FormActions.ts

```diff
@@ -0,0 +1,26 @@
+import Onyx from 'react-native-onyx';
+import {PartialDeep} from 'type-fest';
+import {KeyValueMapping} from 'react-native-onyx/lib/types';
+import * as OnyxCommon from '../../types/onyx/OnyxCommon';
+import {OnyxKey, OnyxValues} from '../../ONYXKEYS';
+
+type KeysWhichCouldBeDraft<T extends keyof OnyxValues> = T extends `${infer U}Draft` ? U : never;
+type DraftKeysWithoutDraftSuffix = KeysWhichCouldBeDraft<keyof OnyxValues>;
+
+function setIsLoading(formID: OnyxKey, isLoading: boolean) {
+    Onyx.merge(formID, {isLoading});
```

Comment:
```suggestion
    Onyx.merge(formID, {isLoading} satisfies Form);
```

The second parameter is being inferred to `unknown`, so we can make it more safe here by making sure it complies to `Form` type.